docs/get-started/clinc150/index.md

```diff
@@ -0,0 +1,843 @@
+# Finetuning a Transformer for Intent Classification
+
+````{info}
+This example builds a transformer model using `pytorch` and the `transformers` package. You can install both packages using:
+
+```
+pip install torch transformers
+```
+````
+
+This example demonstrates how to finetune a model on textual data using `finetuner`.
+Specifically we will tune a transformer model on an Intent Classification task. Intent
+classification is the problem where we try to predict the user intent from a user utterance.
+It is a common step in chatbots and Conversational AI, where after the user speech has been
+decoded to text, we try to represent the meaning of the text symbolically by predicting
+intents and semantic entities. For example:
+
+* I want to book a flight - intent: `book-flight`
+* What is the weather forecast for tomorrow? - intent: `get-weather`
+
+The intent classification task is usually formulated as text classification i.e. we build
+a classifier to predict intents on input text. In this example, we will formulate
+the problem as a search task and use `finetuner` to tune text representations. We will build
+an embedding model that embeds text to a high dimensional space and then we will tune the
+model so that texts that belong to the same class (intent) are represented in proximity
+and texts that belong to separate classes are pulled apart in our embedding space. To 
+convert our embedding model back to a useful intent prediction model, we will implement a
+simple nearest neighbor rule.
+
+
+## CLINC150
+
+We will use the CLINC150 dataset as the base of our experiment. It is a dataset of
+utterance-intent pairs and is commonly used for evaluating intent models. It comes
+in train, val and test splits and contains 150 intents from various chatbot domains.
+For more info on the CLINC150 dataset, check out the
+[dataset repo](https://github.com/clinc/oos-eval).
+
+Firstly, let's clone the dataset locally:
+```bash
+git clone https://github.com/clinc/oos-eval.git
```

Comment:
I think you can provide a link to only the datasets needed to download with `curl`